app/bundles/LeadBundle/Controller/Api/LeadApiController.php

```diff
@@ -661,6 +661,45 @@ public function getEventsAction($id)
         return $this->handleView($this->view($events));
     }
 
+    /**
+     * @param array  $parameters
+     * @param Lead   $entity
+     * @param string $action
+     *
+     * @return mixed|void
+     */
+    protected function prepareParametersForBinding($parameters, $entity, $action)
```

Comment:
`array $parameters`?

Reply:
Not possible because of 
```
PHP Warning - Declaration of Mautic\\LeadBundle\\Controller\\Api\\LeadApiController::prepareParametersForBinding(array $parameters, $entity, $action) should be compatible with Mautic\\ApiBundle\\Controller\\CommonApiController::prepareParametersForBinding($parameters, $entity, $action
```